torch/_dynamo/config.py

```diff
@@ -454,6 +454,8 @@ def default_debug_dir_root():
 # Note: AOT Autograd will still trace joint graphs.
 compiled_autograd = False
 
+enable_compiler_collectives = False
```

Comment:
Add comment.  Should explain what this does and warn about requirement that every rank does the same compiles.